GOVERNANCE.md

```diff
@@ -0,0 +1,87 @@
+
+- **Commitment to the project**:
+  - Participate actively and impactfully in discussions, contributions, code and documentation reviews `this will need quantifying eventually`
+  - Perform useful reviews for a significant number of non-trivial pull requests `this will need quantifying eventually`
+  - Contribute non-trivial pull requests and have them merged `this will need quantifying eventually`
+- Ability to write quality code and/or documentation
+- Ability to collaborate with the team
+- Understanding of how the team works (policies, processes for testing and code review, etc)
+- Understanding of the project's code base and coding and documentation style
+
+A new Maintainer must be proposed by an existing maintainer by opening an issue with the title "New Maintainer Proposal". A 2/3 vote of existing Maintainers approves the application. Maintainers nominations will be evaluated without prejudice to employer or demographics.
+
+Maintainers who are selected will be granted the necessary GitHub rights.
+
+### Removing a Maintainer
+
+Maintainers may resign at any time if they feel that they will not be able to continue fulfilling their project duties.
+
+Maintainers may also be removed after being inactive, failing to fulfill their Maintainer responsibilities, violating the [Code of Conduct](./code-of-conduct.md), or other reasons. Inactivity is defined as a period of very low or no activity in the project for a year or more, with no definite schedule to return to full Maintainer activity.
+
+A Maintainer may be removed at any time by a 2/3 vote of the remaining maintainers.
+
+Depending on the reason for removal, a Maintainer may be converted to **Emeritus** status. Emeritus Maintainers will still be consulted on some project matters and can be rapidly returned to Maintainer status if their availability changes.
+
+## Meetings
+
+Time zones permitting, Maintainers are expected to participate in the ModelPack office hours virtual meeting, which occurs bi-weekly on Thursday at 10:00ET.
+
+### ModelPack Public Office Hours (bi-weekly)
+
+**Every second Thursday @ 14:00 - 15:00**
+**Time zone**: UTC
+**Video call link**: [Zoom](https://zoom.us/j/99127808948?pwd=CggVIWsBPt1jUSbLaWCTcgaHpDB2tl.1)
+**Meeting ID**: 991 2780 8948
+**Passcode**: 644187
+
+Maintainers may also have closed meetings in order to discuss sensitive matters or Code of Conduct violations. Such meetings should be scheduled by any Maintainer on receipt of a CoC report. All current Maintainers must be invited to such closed meetings, except for any Maintainer who is accused of a CoC violation.
+
+## Code of Conduct
+
+Everything we do in the ModelPack community is governed by our [Code of Conduct](./code-of-conduct.md). Violations by community members will be discussed and resolved by the Maintainers in their private Discord channel. If a Maintainer is directly involved in the report, the Maintainers will instead designate two Maintainers to resolve the issue.
```

Comment:
```suggestion
Everything we do in the ModelPack community is governed by our [Code of Conduct](./code-of-conduct.md). Violations by community members will be discussed and resolved by the Maintainers in a private forum. If a Maintainer is directly involved in the report, the Maintainers will instead designate two Maintainers to resolve the issue.
```

Reply:
There is no private maintainers Slack channel. Should a maintainers channel be established?